### ISSUE TITLE
Feature idea: Expose iCals with your extraterrestrial birthdays.

### ISSUE THREAD

**Issue by scrivener** [emilyaviva/extraterrestrial-birthday-calculator#2]
Easiest might be to have an endpoint at GET /calendar/<birthday>/<planet>/ or something that returns an .iCal response? I whipped something up that will generate such a calendar this morning.

Main disadvantage is probably that it's more complicated/burdensome on the server side than hosting a static page, I suppose.

Not sure whether calendar apps/programs keep checking to see whether the URL has updated. And it might be obnoxious to keep getting hit by calendars (especially when a particular calendar never changes...)

**Comment by emilyaviva** [comment]
It could be done. The disadvantage, as you point out, is that this would most likely take the form of an Ajax call, whereas currently the entire app is being run client-side. It would definitely be something neat to implement, though.

**Comment by scrivener** [comment]
Yeah, agreed. I suppose one could also generate a few million static ical files, one for each birthday/planet pair in the last 120 years, and just serve those instead :p

**Comment by emilyaviva** [comment]
Well, I suppose it should be possible to have some kind of iCal-format-event-generating function done on client-side only, I don't see why it wouldn't be possible, but I personally don't know how to write such a thing.

**Comment by scrivener** [comment]
True, but that wouldn't make it possible for a calendar like Google Calendar to import the ical by URL.
